internal/provider/resource_wlan_test.go

```diff
@@ -472,3 +499,34 @@ resource "unifi_wlan" "test" {
 }
 `, vlanID)
 }
+
+func testAccWLANConfig_minimum_data_rate(vlanID int) string {
+	return fmt.Sprintf(`
+data "unifi_ap_group" "default" {
```

Comment:
A minor point but I'd prefer that the Terraform code is formatted as per `terraform fmt`